scripts/install_third_party_libs.py

```diff
@@ -131,9 +137,13 @@ def compile_protobuf_files(proto_files_paths):
     buf_path = os.path.join(
         BUF_DIR,
         BUF_DARWIN_FILES[0] if common.is_mac_os() else BUF_LINUX_FILES[0])
-    for path in proto_files_paths:
-        command = [
-            buf_path, 'generate', path]
+    for path, js_and_ts_files_required in proto_files_paths.items():
+        if js_and_ts_files_required:
```

Comment:
Could you perhaps add a comment here about the differences between the two commands? I don't quite understand the significance of buf_gen_backend.yaml here.